apps/server/src/modules/h5p-editor/service/h5p-player.service.ts

```diff
@@ -1,15 +1,71 @@
-import { H5PConfig, H5PPlayer, UrlGenerator } from '@lumieducation/h5p-server';
+import { H5PConfig, H5PPlayer, UrlGenerator, IPlayerModel } from '@lumieducation/h5p-server';
 
 import { ContentStorage } from '../contentStorage/contentStorage';
 import { LibraryStorage } from '../libraryStorage/libraryStorage';
 
+const renderer = (model: IPlayerModel): string => `<!doctype html>
+<html class="h5p-iframe">
+<head>
+    <meta charset="utf-8">
+    ${model.styles.map((style) => `<link rel="stylesheet" href="${style}"/>`).join('\n    ')}
+    ${model.scripts.map((script) => `<script src="${script}"></script>`).join('\n    ')}
+
+    <script>
+        window.H5PIntegration = ${JSON.stringify(model.integration, null, 2)};
+    </script>
+</head>
+<body>
+    <div class="h5p-content" data-content-id="${model.contentId as string}"></div>
+
+	<script>
+		(() => {
+			let iframeIdentifier = undefined;
+
+			function postSize(identifier) {
+				const height = H5P?.jQuery(".h5p-content")[0].clientHeight;
+
+				if (iframeIdentifier && height) {
+					window.parent.postMessage(JSON.stringify({
+						identifier: identifier,
+						size: {
+							x: document.body.scrollWidth,
+							y: height
+						}
+					}), '*');
+				}
+			}
+
+			window.addEventListener("message", (event) => {
+				try {
+					const message = JSON.parse(event.data);
+					
+					if (message.function === "getSize") {
+						iframeIdentifier = message.identifier;
+						postSize(iframeIdentifier);
+					}
+				} catch (error) {
+					// Ignore message
+				}
+			})
+
+			H5P.externalDispatcher.on("initialized", function() {
+				H5P.instances[0].on('resize', function() {
+					postSize(iframeIdentifier);
+				});
+			});
+		})();
+	</script>
+</body>
+</html>`;
```

Comment:
As previously discussed the H5P player and editor are loaded in an iFrame to prevent document/window-pollution of the client.
H5P globally loads (a lot) of scripts that we probably don't want to be loaded for the entire client.
Serving HTML and displaying that in an iframe circumvents this issue. This is already the way it worked before this PR, this just implements a renderer more suited to our needs instead of the default that Lumi ships with.